From 709fd34ace549713ba2e97476239aef4b7f1baa0 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Wed, 22 Feb 2012 14:12:12 +0100
Subject: [PATCH 056/109] scsi-generic: reenable

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1329919979-20948-56-git-send-email-pbonzini@redhat.com>
Patchwork-id: 37536
O-Subject: [RHEL 6.3 qemu-kvm PATCH v2 055/102] scsi-generic: reenable
Bugzilla: 782029
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

RHEL only.
---
 Makefile.objs |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 Makefile.objs |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.objs b/Makefile.objs
index 80772a6..c538b25 100644
--- a/Makefile.objs
+++ b/Makefile.objs
@@ -87,7 +87,7 @@ common-obj-y += eeprom93xx.o
 common-obj-y += cdrom.o
 common-obj-y += usb.o usb-hub.o usb-$(HOST_USB).o usb-hid.o
 common-obj-y += usb-bus.o usb-desc.o
-common-obj-y += usb-msd.o scsi-bus.o scsi-disk.o
+common-obj-y += usb-msd.o scsi-bus.o scsi-disk.o scsi-generic.o
 common-obj-$(CONFIG_SSI) += ssi.o
 common-obj-$(CONFIG_SSI_SD) += ssi-sd.o
 common-obj-$(CONFIG_SD) += sd.o
-- 
1.7.7.6

